Core/include/Acts/TrackFinder/CombinatorialKalmanFilter.hpp

```diff
@@ -1181,4 +1181,4 @@ class CombinatorialKalmanFilter {
 
 };  // namespace Acts
 
-}  // namespace Acts
\ No newline at end of file
+}  // namespace Acts
```

Comment:
? Only change here?

Reply:
Missing newline at the end of the file was added.

Reply:
Right, but why? This is the only change in that file.

Reply:
Auto-clang-format. I can remove it.

Reply:
Has been removed.